src/components/LeaderBoard/LeaderBoard.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styles from './LeaderBoard.module.css';
+import history from '../../services/history';
+import { setPlayers } from './LeaderBoardActions';
+
+export class LeaderBoard extends React.Component {
+    componentDidMount() {
+        this.props.setPlayers();
```

Comment:
Should this be `getPlayers` or `fetchPlayers`?

Reply:
sure thaaang